src/polytopes/simplex.jl

```diff
@@ -37,3 +37,21 @@ nvertices(::Type{<:Simplex{K}}) where {K} = K + 1
 
 Base.isapprox(s₁::Simplex, s₂::Simplex; kwargs...) =
   all(isapprox(v₁, v₂; kwargs...) for (v₁, v₂) in zip(vertices(s₁), vertices(s₂)))
+
+"""
+    normal(splx::Simplex)
+
+Compute normal vector to simplex with euclidean norm of 1.
+"""
+function normal(splx::Simplex{K,Dim,T}) where {K,Dim,T<:Real}
+  Dim > K || throw(
+    ArgumentError("simplex embedding dimension must be larger than parametric dimension for a normal vector to exist")
+  )
+  # It turns out the QR decomposition can be used to find an orthogonal basis,
+  # where by construction the nth column of `Q` is orthogonal to columns 1:(n-1).
+  # Further, `qr` guarantees `Q` to be unitary, therefore `norm(Q[:, i]) ≈ 1` for all columns.
+  p0, pothers... = vertices(splx)
+  extendedbasis = [(p - p0 for p in pothers)... rand!(similar(coordinates(p0)))]
+  normal = qr(extendedbasis).Q[:, end]
+  return normal
```

Comment:
no explicit return

Reply:
Done. The suggested style is
```julia
function foo()
  retval = 1+2
  retval
end
```

or

```julia
function foo()
  retval = 1+1
end
```

or 

```julia
function foo()
  1+2
end
```
?

I went with the first, because it allows to assign a name to the computed variable (helpful for reading). Let me know if you prefer otherwise.

Reply:
The last option is preferred. But this is assessed on a case-by-case basis. The `return` is avoided as much as possible.